File: 1012.py
a = input().split()

A = float(a[0])
B = float(a[1])
C = float(a[2])

pi = 3.14159

print('TRIANGULO: {:.3f}'.format((A*C) / 2))
print('CIRCULO: {:.3f}'.format(pi*(C**2)))
print('TRAPEZIO: {:.3f}'.format(((A+B)*C)/2))
print('QUADRADO: {:.3f}'.format(B**2))
print('RETANGULO: {:.3f}'.format(A*B))

File: 1012.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
    
var a = lines.shift().split(' ');

var A = parseFloat(a[0]);
var B = parseFloat(a[1]);
var C = parseFloat(a[2]);

var pi = 3.14159;

console.log('TRIANGULO: ' + ((A*C) / 2).toFixed(3));
console.log('CIRCULO: ' + (pi*(C**2)).toFixed(3));
console.log('TRAPEZIO: ' + (((A+B)*C)/2).toFixed(3));
console.log('QUADRADO: ' + (B**2).toFixed(3));
console.log('RETANGULO: ' + (A*B).toFixed(3));